Rubberduck.Refactorings/ExtractInterface/ExtractInterfaceRefactoring.cs

```diff
@@ -135,5 +136,36 @@ private string GetInterfaceModuleBody(ExtractInterfaceModel model)
             DeclarationType.Document,
             DeclarationType.UserForm
         };
+
+        public static bool CanExecute(RubberduckParserState state, QualifiedModuleName qualifiedName)
```

Comment:
I'd prefer if this weren't static. That simplifies testing the usage spaces, because it allows for mocking. Considering that all usages have an instance of the refactoring available, that shouldn't be much of an issue to change as well :)